### PR TITLE
[feature]: kick-off project

### PR DESCRIPTION
**Pull request by yovanny-wolox**
## Summary
- kick-off project

## Known Issues
None

## Trello Card
https://trello.com/c/MLk8gPy9/1-kick-off

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+language: node_js
+
+node_js:
+  - "10.14.1"
```
Comment (marioportela1993) [inline .travis.yml]: I recommend to you to use the  14.xx.xx  node version :)

File: .nvmrc [hunk 2]
```diff
@@ -0,0 +1 @@
+10.14.1
```
Comment (marioportela1993) [inline .nvmrc]: Change here the version to 14.xx.xxx

File: LICENSE.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2019 Wolox
```
Comment (marioportela1993) [inline LICENSE.md]: Please change 2019 -> 2021

File: README copy.md [hunk 4]
```diff
@@ -0,0 +1,160 @@
...
+
+This project is maintained by [Wolox](https://github.com/wolox) and it was written by [Wolox](http://www.wolox.com.ar).
+
+![Wolox](https://raw.githubusercontent.com/Wolox/press-kit/master/logos/logo_banner.png)
+
+## License
+
+**WTraining** is available under the MIT [license](LICENSE.md).
+
+    Copyright (c) 2019 Wolox
```
Comment (marioportela1993) [inline README copy.md]: Change 2019 -> 2021

File: package.json [hunk 5]
```diff
@@ -0,0 +1,111 @@
+{
+  "name": "yl-express-js",
+  "version": "0.1.0",
+  "description": "WTraining",
+  "engines": {
+    "node": "10.14.1",
```
Comment (marioportela1993) [inline package.json]: Change the node version as above comments

File: package.json [hunk 6]
```diff
@@ -0,0 +1,111 @@
...
+    "npm": "6.4.1"
```
Comment (marioportela1993) [inline package.json]: Same for npm version

**Review by marioportela1993 — CHANGES_REQUESTED** [review]
Excellent job Yova, only a few comments, check them and let's me know if you have questions

File: .travis.yml (hunk 1 above)
Comment (yovanny-wolox) [reply .travis.yml]: I aggre. Done!

File: .nvmrc (hunk 2 above)
Comment (yovanny-wolox) [reply .nvmrc]: I aggre. Done!

File: LICENSE.md (hunk 3 above)
Comment (yovanny-wolox) [reply LICENSE.md]: Done!

File: README copy.md (hunk 4 above)
Comment (yovanny-wolox) [reply README copy.md]: Done!

File: package.json (hunk 5 above)
Comment (yovanny-wolox) [reply package.json]: Done!

File: package.json (hunk 6 above)
Comment (yovanny-wolox) [reply package.json]: Done!

**Review by marioportela1993 — APPROVED** [review]

**Pull request merged**